.env

```
@@ -0,0 +1 @@
+VITE_API_KEY=f7c82d8d69637b69b5e8841185b42153
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)